src/main/java/com/amihaiemil/versioneye/MkJsonServer.java

```diff
@@ -54,22 +54,13 @@ public MkJsonServer() {
         MkJsonServer.initServer(this.storage);
     }
     
-    /**
-     * Ctor.
-     * @param storage JsonObjectBuilder which holds all the mock VersionEye
-     * data.
-     */
-    public MkJsonServer(final NfJsonObjectBuilder storage) {
```

Comment:
@SherifWaly I removed this Ctor, since the users will not know how to use NfJsonObjectBuilder and how to properly init the server, in what format... and they won't need this anyway. Anything that needs initialized in the server should be through methods :)